draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -251,6 +253,56 @@ the translator may need to rewrite the RTP packets to fit into the smaller MTU
+| [3.2.2](https://datatracker.ietf.org/doc/html/rfc7667#section-3.2.2) | Topo-Back-To-Back | yes  | Note-SEC <br> Note-MTU <br> Note-MCast|
+| [3.3.1](https://datatracker.ietf.org/doc/html/rfc7667#section-3.3.1) | Topo-ASM | no | Note-MCast |
+| [3.3.2](https://datatracker.ietf.org/doc/html/rfc7667#section-3.3.2) | Topo-SSM | partly | Note-MCast <br> Note-UCast-MCast |
+| [3.3.3](https://datatracker.ietf.org/doc/html/rfc7667#section-3.3.3) | Topo-SSM-RAMS | partly | Note-MCast <br> Note-MCast-UCast |
+| [3.4](https://datatracker.ietf.org/doc/html/rfc7667#section-3.4) | Topo-Mesh | yes | Note-MCast |
+| [3.5.1](https://datatracker.ietf.org/doc/html/rfc7667#section-3.5.1) | Topo-PtM-Trn-Translator | possibly | Note-MCast <br> Note-MTU <br> Note-Topo-PtM-Trn-Translator |
+| [3.6](https://datatracker.ietf.org/doc/html/rfc7667#section-3.6) | Topo-Mixer | possibly | Note-MCast <br> Note-Topo-Mixer|
+| [3.6.1](https://datatracker.ietf.org/doc/html/rfc7667#section-3.6.1) | Media-Mixing-Mixer | partly | Note-Topo-Mixer |
+| [3.6.2](https://datatracker.ietf.org/doc/html/rfc7667#section-3.6.2) | Media-Switching-Mixer | partly | Note-Topo-Mixer |
+| [3.7](https://datatracker.ietf.org/doc/html/rfc7667#section-3.7) | Selective Forwarding Middlebox | yes | Note-MCast <br> Note-Topo-Mixer |
+| [3.8](https://datatracker.ietf.org/doc/html/rfc7667#section-3.8) | Topo-Video-switch-MCU | yes  |  Note-MTU <br> Note-MCast <br> Note-Topo-Mixer |
+| [3.9](https://datatracker.ietf.org/doc/html/rfc7667#section-3.9) | Topo-RTCP-terminating-MCU | yes | Note-MTU <br> Note-MCast <br> Note-Topo-Mixer |
+| [3.10](https://datatracker.ietf.org/doc/html/rfc7667#section-3.10) | Topo-Split-Terminal | yes | Note-MCast|
+| [3.11](https://datatracker.ietf.org/doc/html/rfc7667#section-3.11) | Topo-Asymmetric | Possibly | Note-Warn, <br> Note-MCast, <br> Note-MTU |
+
+Note-NAT:
+: QUIC {{!RFC9000}} doesn't support NAT traversal.
+
+Note-MTU:
+: Supported, but may require MTU adaptation.
+
+Note-Sec:
+: Note that RTP-over-QUIC provides mandatory security, and other RTP transports do not. {{sec-considerations}} describes strategies to prevent the inadvertent disclosure of RTP sessions to unintended third parties.
+
+Note-MCast:
+: QUIC {{!RFC9000}} cannot be used for multicast paths.
+
+Note-UCast-MCast:
+: The topology refers to a *Distribution Source*, which receives and relays RTP from a number of different media senders via unicast before relaying it to the receivers via multicast. QUIC can be used between the senders and the *Distribution Source*, if the *Distribution Source* is not just a logical entity but a real node.
+
+Note-MCast-UCast:
+: The topology refers to a *Burst Source* or *Retransmission Source*, which retransmits RTP to receivers via unicast. QUIC can be used between the *Retransmission Source* and the receivers, if the *Distribution Source* is not just a logical entity but a real node.
```

Comment:
I think this was meant to say *Burst Source* instead of *Distribution Source*. But I just re-read the text about SSM in RFC 7667, and I think we can actually drop this last part of the sentence here and also for `Note-UCast-Mcast`. I don't know why I thought *Distribution Source* and *Burst Source* could just be *logical entities*. The text says:

> These sources send media to a dedicated Distribution Source, which forwards the RTP streams to the multicast group on behalf of the original RTP sources.

Sorry, I didn't catch that earlier!